helsingborg-stad/app-mitt-helsingborg
# refactor(images): refactor images catalog in source

## Explain the changes you’ve made
Removed images catalog under source

## Explain why these changes are made
These files weren't used. Those who were used were moved to the images catalog under source/assets

## How to test

1. Checkout this branch
2. Fire upp the simulator by running the command `yarn ios` or `yarn android`
3. Run Grundansökan form and see if anything breakes.

## Tested environments

- [] Storybook on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.
